shoplist/src/main/java/com/netguru/codereview/network/ShopListRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network
+
+class ShopListRepository(private val shopListApi: ShopListApi) {
+
```

Comment:
if we use Dagger it will be good if we provide this contractor using it
inject ShopListApi